x/evidence/keeper/keeper.go

```diff
@@ -165,3 +180,7 @@ func (k Keeper) MustMarshalEvidence(evidence exported.Evidence) []byte {
 
 	return bz
 }
+
+func (k Keeper) GetCodec() types.Codec {
+	return k.cdc
```

Comment:
If you're going to allow this, then you cannot have the concrete codec be a reference for safety reasons.

Reply:
Is the reason for this that other module keeper's could mess with the codec internals?